<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TouhouSatsu</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart_Game"/>
    <addaction name="actionStart_Server"/>
    <addaction name="actionPC_Console_Start"/>
    <addaction name="actionReplay"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionGeneral_Overview"/>
    <addaction name="actionCard_Overview"/>
    <addaction name="actionView_ban_list"/>
    <addaction name="actionBroadcast"/>
    <addaction name="actionReplay_file_convert"/>
    <addaction name="actionRecord_analysis"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_Hotkey"/>
    <addaction name="actionNever_nullify_my_trick"/>
    <addaction name="actionView_Discarded"/>
    <addaction name="actionView_distance"/>
    <addaction name="actionServerInformation"/>
    <addaction name="actionSurrender"/>
    <addaction name="actionSaveRecord"/>
    <addaction name="separator"/>
    <addaction name="actionDownload_Hero_Skin_and_BGM"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Us"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_GPLv3"/>
    <addaction name="actionAbout_Lua"/>
    <addaction name="separator"/>
    <addaction name="actionAcknowledgement_2"/>
    <addaction name="separator"/>
    <addaction name="actionRole_assign_table"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Hide_Menu"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionMinimize_to_system_tray"/>
   </widget>
   <widget class="QMenu" name="menuCheat">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Cheat</string>
    </property>
    <addaction name="actionDamage_maker"/>
    <addaction name="actionDeath_note"/>
    <addaction name="actionRevive_wand"/>
    <addaction name="actionExecute_script_at_server_side"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuCheat"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../image/system/settings.png</normaloff>../image/system/settings.png</iconset>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionStart_Server">
   <property name="text">
    <string>Start server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStart_Game">
   <property name="text">
    <string>Start game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGeneral_Overview">
   <property name="text">
    <string>General overview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCard_Overview">
   <property name="text">
    <string>Card overview</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEnable_Hotkey">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable hotkey</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionView_Discarded">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View Discarded</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionView_distance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View distance</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionServerInformation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Server information</string>
   </property>
  </action>
  <action name="actionReplay">
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionSaveRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save battle record</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionShow_Hide_Menu">
   <property name="text">
    <string>Show/Hide menu</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionSurrender">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Surrender</string>
   </property>
  </action>
  <action name="actionMinimize_to_system_tray">
   <property name="text">
    <string>Minimize to system tray</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionRole_assign_table">
   <property name="text">
    <string>Role assign table</string>
   </property>
  </action>
  <action name="actionBroadcast">
   <property name="text">
    <string>Broadcast ...</string>
   </property>
  </action>
  <action name="actionDamage_maker">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Damage maker</string>
   </property>
  </action>
  <action name="actionDeath_note">
   <property name="text">
    <string>Death note ...</string>
   </property>
  </action>
  <action name="actionRevive_wand">
   <property name="text">
    <string>Revive wand ...</string>
   </property>
  </action>
  <action name="actionPC_Console_Start">
   <property name="text">
    <string>PC Console Start</string>
   </property>
  </action>
  <action name="actionReplay_file_convert">
   <property name="text">
    <string>Replay file convert ...</string>
   </property>
  </action>
  <action name="actionView_ban_list">
   <property name="text">
    <string>View ban list</string>
   </property>
  </action>
  <action name="actionAbout_Lua">
   <property name="text">
    <string>About Lua</string>
   </property>
  </action>
  <action name="actionExecute_script_at_server_side">
   <property name="text">
    <string>Execute script at server side ...</string>
   </property>
  </action>
  <action name="actionRecord_analysis">
   <property name="text">
    <string>Record analysis</string>
   </property>
  </action>
  <action name="actionAbout_GPLv3">
   <property name="text">
    <string>About GPLv3</string>
   </property>
  </action>
  <action name="actionNever_nullify_my_trick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Never nullify my trick</string>
   </property>
  </action>
  <action name="actionAcknowledgement_2">
   <property name="text">
    <string>Acknowledgement</string>
   </property>
  </action>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionDownload_Hero_Skin_and_BGM">
   <property name="text">
    <string>Download/Update contents</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
